arcgis-ios-sdk-samples/Route & Navigation/Offline routing/OfflineRoutingViewController.swift

```diff
@@ -299,11 +299,11 @@ class OfflineRoutingViewController: UIViewController, AGSGeoViewTouchDelegate {
     
     //MARK: toggle details view
     
-    private func toggleDetailsView(on: Bool) {
+    fileprivate func toggleDetailsView(_ on: Bool) {
```

Comment:
similar to other comments above about having argument name to clarify call site usage